app/src/main/java/org/thoughtcrime/securesms/webrtc/PeerConnectionWrapper.kt

```diff
@@ -73,7 +73,10 @@ class PeerConnectionWrapper(private val context: Context,
         newPeerConnection.setAudioPlayout(true)
         newPeerConnection.setAudioRecording(true)
 
-        newPeerConnection.addStream(mediaStream)
+        // Calls to `addStream` are deprecated & cause errors so we must use `addTracks` when
+        // using `io.github.webrtc-sdk:android:114.5735.10` and newer.
+        newPeerConnection.addTrack(mediaStream.audioTracks[0])
```

Comment:
Should we / can we check that there is an audio track (and a video track, below) before adding it? Like, if a user has their microphone muted we might not get one perhaps?